app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php

```diff
@@ -418,7 +377,7 @@ public function uploadShapefile(Request $request)
                 $polygonLoadedList = isset($body['polygon_loaded']) && filter_var($body['polygon_loaded'], FILTER_VALIDATE_BOOLEAN);
                 $submitPolygonsLoaded = $request->input('submit_polygon_loaded') === true || $request->input('submit_polygon_loaded') === 'true';
                 if (! $polygonLoadedList && ! $submitPolygonsLoaded) {
-                    $uuid = $this->insertGeojsonToDB($geojsonFilename, $site_id, 'site', $body['primary_uuid'] ?? null);
+                    $job = new InsertGeojsonToDBJob($geojsonFilename, $site_id, 'site', $body['primary_uuid'] ?? null);
```

Comment:
Remember that in AWS, the workers and the web boxes are separate EC2 machines and do not share a file system. This geoJson file is going to need to get uploaded to an S3 bucket and then pulled down in the job.